src/js/control-bar/control-bar.js

```diff
@@ -56,7 +55,8 @@ class ControlBar extends Component {
 ControlBar.prototype.options_ = {
   children: [
     'playToggle',
-    'volumeMenuButton',
+    'muteToggle',
```

Comment:
I think we should keep mutetoggle and volumecontrol working as is if they're used by themselves.